app/labor/cache_buster.rb

```diff
@@ -19,6 +19,8 @@ def self.bust(path)
     #
     # https://github.com/fastly/fastly-ruby#efficient-purging
     return unless Rails.env.production?
+    return if ENV["FASTLY_API_KEY"].blank?
+    return if ENV["FASTLY_API_KEY"] == "foobarbaz"
```

Comment:
is there a reason we're hardcoding this `foobarbaz` string 🤣 ?

Reply:
I can just unset the `FASTLY_API_KEY` or totally remove them so we are not hardcoding things. That might be the better path here.

Reply:
I don't know that we want to return without fastly set because busting with nginx will involve changes to CacheBuster and the Purgeable concern and if we short-circuit without fastly, then nginx won't work as the default in the fallback? If we did this, then I suspect that we'll have to do a bunch of conditional checks to see if nginx is "configured" which maybe is fine eventually, but for now, until we have nginx work done, I wonder if we actually really need to do these short-circuits at all?

Reply:
Wanted to also add this TL;DR: unless there is a strong reason to add these checks (am I maybe missing some important piece of info? Let me know if so!), I'd vote to not short-circuit/check for fastly ENV vars.

I'll be in this code soon in order to handle nginx purging, and IMHO, that seems like a better time to figure out ~whether we want to do~ how to handle these checks. :)